TYPO3.Media/Classes/TYPO3/Media/Domain/Model/ThumbnailConfiguration.php

```diff
@@ -212,7 +212,7 @@ protected function logDeprecation()
     {
         if (!static::$loggedDeprecation) {
             static::$loggedDeprecation = true;
-            $this->logger->log('TYPO3.Media is configured to simulate the deprecated Neos 1.2 behaviour. Please check the setting "TYPO3.Media.behaviourFlag".',
+            $this->logger->log('Neos.Media is configured to simulate the deprecated Neos 1.2 behaviour. Please check the setting "Neos.Media.behaviourFlag".',
```

Comment:
The will log a non-existing package key (for now), so don't merge without actual vendor name change.